src/item.cpp

```diff
@@ -6406,3 +6412,18 @@ void item::set_birthday( const time_point bday )
 {
     this->bday = bday;
 }
+
+int check_freezing_temperature( const item &it )
+{
+    int temp = FREEZING_TEMPERATURE;
+    if( it.made_of( material_id( "alcohol" ) ) && it.made_of( material_id( "water" ) ) ) {
+        temp = 23; // -5C, wine, beer etc. (assumption: 10% ethanol) 
+    }
+    if( it.made_of( material_id( "alcohol" ) ) && !it.made_of( material_id( "water" ) ) ) {
```

Comment:
This can be boiled down to:

```c++
if( it.made_of( material_id( "alcohol" ) ) {
    temp = it.made_of( material_id( "water" ) ) ? 23 : -22;
}
```
or (assuming it's a member function):
```c++
if( made_of( material_id( "alcohol" ) ) {
    temp = made_of( material_id( "water" ) ) ? 23 : -22;
}
```

Reply:
Ok, thats indeed simplier.